Tools-Override/Build.Common.props

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- 
+     This file will contain all of the common properties from most repos. The intention is to only have 
+     repo specific properties inside the repos, and move to this file everything that is common.
+  -->
+
+  <!--
+    Import the reference assembly props
+    
+      Sets Properties:
+        IsReferenceAssembly - Set if the project is in the ref assm path
+  -->
+  <Import Project="$(MSBuildThisFileDirectory)ReferenceAssemblies.props" Condition="'$(ExcludeReferenceAssembliesImport)'!='true'" />
+
+  <PropertyGroup>
+    <CompilerResponseFile Condition="'$(CheckSumSHA256)'!='false'">$(MSBuildThisFileDirectory)checksum.rsp</CompilerResponseFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GetNuGetPackageVersionsDependsOn>CreateVersionFileDuringBuild</GetNuGetPackageVersionsDependsOn>
+  </PropertyGroup>
+
+
+  <PropertyGroup Condition="'$(___BuildConfiguration)'==''">
```

Comment:
Please file a follow-up issues for cleaning up these targets/props as I suspect a number of them are no longer interesting.